components/feature/autofill/src/test/java/mozilla/components/feature/autofill/test/MockStructure.kt

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.autofill.test
+
+import android.view.autofill.AutofillId
+import mozilla.components.feature.autofill.handler.FillRequestHandlerTest
+import mozilla.components.feature.autofill.structure.AutofillNodeNavigator
+import mozilla.components.feature.autofill.structure.RawStructure
+import java.io.File
+
+internal fun FillRequestHandlerTest.createMockStructure(filename: String, packageName: String): RawStructure {
+    val classLoader = javaClass.classLoader ?: throw RuntimeException("No class loader")
+    val resource = classLoader.getResource(filename) ?: throw RuntimeException("Resource not found")
+    val file = File(resource.path)
+
+    return object : RawStructure {
```

Comment:
Why an anonymous object, and not, say, a `TestRawStructure`? Similarly, `AssistStructure.toRawStructure` could return something like `AndroidRawStructure` or something.

That could make stacktraces a little bit more friendly?